scripts/download.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/sh
+
+# Usage:
+#   curl ... | ENV_VAR=... sh -
+#       or
+#   ENV_VAR=... ./setup.sh
+#
+# Examples:
+#   Downloading scripts from a "develop" branch in a temp local folder "temp":
+#     curl -sfL https://raw.githubusercontent.com/SUSE/lab-setup/feature/init-solution/scripts/setup.sh | GIT_REVISION=refs/heads/develop sh -s -- -o temp
+#   Downloading scripts from a specific revision "d8b7564fbf91473074e86b598ae06c7e4e522b9f" in the default local folder:
+#     curl -sfL https://raw.githubusercontent.com/SUSE/lab-setup/feature/init-solution/scripts/setup.sh | GIT_REVISION=d8b7564fbf91473074e86b598ae06c7e4e522b9f sh -
+#   Testing locally the setup script:
+#     GIT_REVISION=refs/heads/feature/init-solution ./lab-setup/scripts/setup.sh -o temp
+#
+# Environment variables:
+#   - GIT_REVISION
+#     Git revision (refs/heads/<branch-name>, refs/tags/vX.Y.Z for a tag, xxxxxxxxxxxxxxxx for a commit hashcode)
+#   - OUTPUT_FOLDER
+#     Output folder, where the scripts folder will be created with script directory tree inside, overriden if -o is used
+
+info() {
+  echo '[INFO] ' "$@"
+}
+
+warn() {
+  echo '[WARN] ' "$@" >&2
+}
+
+fatal() {
+  echo '[ERROR] ' "$@" >&2
+  exit 1
+}
+
+verify_system() {
+  info 'Verify system requirements'
+  if [ -x /usr/bin/git ] || type git > /dev/null 2>&1; then
```

Comment:
Maybe we should also verify `jq` present as a lot of commands depend on it.

Reply:
Done. Thanks